README.md

```diff
@@ -74,6 +74,70 @@ perfplot log_Array1k_19-09-2019_12-13-49
 
 This will generate a file called `log_Array1k_19-09-2019_12-13-49.pdf`.
 
+# Save results to SQL database
+
+## Requirements
+***ODB 2.5.0***
+> Note: 2.5.0 is a beta version, but it is required if you are using Ubuntu 18.04 LTS and
+gcc 7.4.0, if you are using earlier ubuntu/gcc versions, proceed to the installation of ODB 2.4.0
+below.
+
+Follow the instructions [here](https://www.codesynthesis.com/products/odb/doc/install-build2.xhtml)
+ to install build2 toolchain, odb compiler and runtime libraries.
+
+Required runtime libraries are: libodb, libodb-sqlite/libodb-pgsql/libodb-mysql (depending on which
+ database you want to use) and libodb-boost.
+
+Remember to install the correct gcc plugin, for example (for gcc 7):
+```
+sudo apt-get update
+sudo apt-get install gcc-7-plugin-dev
+```
+Additionally, if you want to use MYSQL database, you will need to build a not yet released version,
+so instead of executing:
+```
+bpkg build libodb-mysql
+bpkg install libodb-mysql
+```
+you will need to execute:
+```
+bpkg build libodb-mysql@https://git.codesynthesis.com/odb/libodb-mysql.git#fix-bind-decl
+bpkg install libodb-mysq
+```
+After installing all the libraries make sure to add `/usr/local/lib` to your PATH:
+```
+export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib
+```
+
+***ODB 2.4.0:***
+All the downloads can be found [here](https://www.codesynthesis.com/products/odb/download.xhtml).
+* Odb compiler
+* Common Runtime Library: libodb-2.4.0
+* Database Runtime Library: libodb-sqlite-2.4.0/libodb-mysql-2.4.0/libodb-pgsql-2.4.0
+* Profile Libraries: libodb-boost-2.4.0, libodb-qt-2.4.0
+
+Follow the [instructions](https://www.codesynthesis.com/products/odb/doc/install-unix.xhtml) to
+install all the components.
+
+## How to build and run
+```
+source ros2_install_path/setup.bash
+mkdir -p perf_test_ws/src
+cd perf_test_ws/src
+git clone https://github.com/ApexAI/performance_test.git
+cd ..
+colcon build --cmake-clean-cache --cmake-args -DCMAKE_BUILD_TYPE=Release -DODB_FOR_SQL_ENABLED=ON
+source install/setup.bash
+ros2 run performance_test perf_test -c ROS2 -l log -t Array1k --max_runtime 10
+```
+
+The default name of resulting database is "odb_test", you can change it by using `--db_name`
```

Comment:
@monidzik The performance test runs now. How do I view the database now? Can you please add instructions on how to setup the DB (probably can put the link of the site) and how to view the tables?

Reply:
@monidzik :
As per offline discussion : I tried the following :
1. There was a `db_name` file that got generated in my `perf_test_ws` after I followed steps upto https://github.com/ApexAI/performance_test/pull/84/files#diff-04c6e90faac2675aa89e2176d2eec7d8R131 using the default database `sqlite`
2. Followed the following to install `sqlitebrowser` on ubuntu 16.04: 
- sudo apt-get update
- sudo apt-get install sqlitebrowser
3. I tried to drag and drop the `db_name file` in the sqlitebrowser application.
Its not showing me the `db_name` folder anymore when I browse to select the daa
![image](https://user-images.githubusercontent.com/15241291/65790577-c6a7ec00-e114-11e9-81e9-f90c24c54137.png)

Reply:
Update: Tried the following and it worked :

1. cd to directory which contains the database file `db_name`
2. Run `sqlitebrowser db_name` and it should open the following window :
![image](https://user-images.githubusercontent.com/15241291/65791379-6f0a8000-e116-11e9-8016-b888a17c134b.png)
Showing the `Analysis Result` and `Experiment Configuration` files.

Reply:
@monidzik Were you able to include these instructions to view in the `README.md`?

Reply:
@daggarwa I don't think it's necessary, it's not directly related to odb and creating sql database - if you already have one, there are many options how to view it, you can just google it. That's why there is no instructions here also how to connect to MySQL or PostreSQL server.